### PR TITLE
chore: add initial files

### PR DESCRIPTION
**Pull request by FaGru3n**
#### What you changed / added?
- added inital files for Catena-X NG 
- added first draft for the `profile/README.md`
#### Why?
- https://github.com/eclipse-tractusx/sig-infra/issues/38
#### Verification
- looked over https://github.com/catenax-ng/foss-example for default files we need for catena-x repositories
#### Additional information
Linking to which files is a bit unclear for me or which repo's should be available within catena-x ng

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Welcome to Catena-X NG 🙌
+
+- [Catena-X (ng) Project page](https://catenax-ng.github.io/)
+- [Eclipse Foundation Tractus-X Project page](https://projects.eclipse.org/projects/automotive.tractusx)
+
+For the latest Tractus-X release, see the [Changelog](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md) of our release repository [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release).
```
Comment (carslen) [inline profile/README.md:6]: Not sure if it's relevant for Catena-X NG to reference to Tractusx Release Changelog. It doesn't hurt, but for me it's irrelevant here. Opinions?

Comment (FaGru3n) [reply profile/README.md:6]: I think your right, maybe a PR as Draft would be better for the next time.

File: profile/README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
-# Welcome to Catenax-ng 🙌
+# Welcome to Catena-X NG 🙌
 
-You can find our support landing page at [https://catenax-ng.github.io/](https://catenax-ng.github.io/)
+For further information check out our Project Page or our pinned repositories:
+- [Catena-X (ng) Project page](https://catenax-ng.github.io/)
+
+Catena-X is powered by Software from the [Eclipse Foundation Tractus-X Project](https://projects.eclipse.org/projects/automotive.tractusx)
```
[the remark this answers is not in the document]
Comment (Siegfriedk) [reply profile/README.md:6]: mmhh: Catena-X is guiding and sponsoring development of EF Tractus-X?

But i'm not very good in this, thats just my guess, you might want to cliarify it with Daniel Miehle

[the remark this answers is not in the document]
Comment (FaGru3n) [reply profile/README.md:6]: new suggestion 
> The Catena-X ng GitHub Organization is part of the Ecosystem of Catena-X and the Catena-X ecosystem is guiding and sponsoring the development of the Eclipse Tractus-X™ under the umbrella of the Eclipse Foundation.

**Review by almadigabor — APPROVED** [review]
LGTM

**Pull request merged**